vdom/tests/test_core.py

```diff
@@ -26,6 +26,18 @@ def test_to_html_unicode():
 def test_to_html_escaping():
     assert div(p("Hello world<script>evil</script>", title='something')).to_html() == '<div><p title="something">Hello world&lt;script&gt;evil&lt;/script&gt;</p></div>'
 
+def test_css():
+    assert div(
+        p('Hello world'),
+        style={
+            'background-color': 'pink',
+            'color': 'white',
+            # Quotes should be entity escaped
+            'font-family': "'something something'"
```

Comment:
Oops. I just realized this is wrong, when used within the notebook. On the VDOM spec, these have to match the DOM attribute names which are camelCase.

The `to_html` is working, the in-notebook version is not.

Reply:
@rgbkrk this is a consequence (sortof) of https://github.com/nteract/vdom/issues/31 right? so the 'right' thing to do is regardless of input, the output is camelCase for the VDOM JSON and kebab case for html generation. Is that right?

Reply:
Or to begin with, let's canonicalize on camelCase, and convert to kebab case for HTML output only?

Reply:
> regardless of input, the output is camelCase for the VDOM JSON and kebab case for html generation. Is that right?

That's correct.

> let's canonicalize on camelCase, and convert to kebab case for HTML output only?

Yes please!